<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1416</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/*QWdget#widget{
 border:2px solid rgb (182, 115, 50);
 border-radius:20px;

}*/
background-color:white; /* Change the background color to green */

  color: #rgb(182, 115, 50);

font-family: &quot;Roboto&quot;, sans-serif; /* Police Roboto */
  padding: 10px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  border-radius: 20px !important; /* Adjust this value to make the corners more rounded */
  cursor: pointer;
</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1421</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:white; /* Change the background color to green */

  color: #rgb(182, 115, 50);

font-family: &quot;Roboto&quot;, sans-serif; /* Police Roboto */
 /* padding: 10px 20px;*/
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
  /* border-radius: 20px !important; /* Adjust this value to make the corners more rounded */
  cursor: pointer;
</string>
    </property>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#supprimer{
background-color:rgb(30, 15,28);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficher">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#afficher{
background-color:rgb(30, 15,28);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>afficher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#modifier{
background-color:rgb(30, 15,28);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#ajouter{
background-color:rgb(30, 15,28);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trier">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>trier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>pdf</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rechercher">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgb(255, 170, 0);
}
</string>
     </property>
     <property name="placeholderText">
      <string>tapez ici</string>
     </property>
    </widget>
    <widget class="QTableView" name="affiche_tab">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>711</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView {
    background-color: transparent;
    border: #000000;
    selection-background-color: #000000;
    selection-color: white;
    color: #000000;
    border: 1px solid #000000;
}

QHeaderView::section {
    background-color: transparent;
    color: #000000;
    padding: 4px;
    border: none;
    border: 1px solid #000000;
}

QTableView::item:selected {
    background-color: #000000;
    color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="stat">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>stat</string>
     </property>
    </widget>
    <widget class="QLabel" name="qr_code">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>10</y>
       <width>331</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: transparent;
    border: #000000;
    selection-background-color: #000000;
    selection-color: white;
    color: #000000;
    border: 1px solid #000000;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="qrcode_2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>30</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>qrcode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_stat">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>461</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: transparent;
    border: #000000;
    selection-background-color: #000000;
    selection-color: white;
    color: #000000;
    border: 1px solid #000000;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="categorie">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>181</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgb(255, 170, 0);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Catégorie</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qrcode">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>181</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgb(255, 170, 0);
}
</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string comment="Nom de Produit"/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Qr code</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="quantite">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>181</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgb(255, 170, 0);
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Qantité</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>181</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color:rgb(255, 170, 0);
}
</string>
     </property>
     <property name="inputMask">
      <string comment="catégorie"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Nom Produit</string>
     </property>
    </widget>
    <widget class="QTableView" name="historique_view">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>360</y>
       <width>581</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView {
    background-color: transparent;
    border: #000000;
    selection-background-color: #000000;
    selection-color: white;
    color: #000000;
    border: 1px solid #000000;
}

QHeaderView::section {
    background-color: transparent;
    color: #000000;
    padding: 4px;
    border: none;
    border: 1px solid #000000;
}

QTableView::item:selected {
    background-color: #000000;
    color: white;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf_2">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(0, 0, 0);

  text-align: center;
  color: white;}</string>
     </property>
     <property name="text">
      <string>pdf historique</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1416</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
